modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -618,6 +621,45 @@ Given the size of your ownCloud filesystem, this may take some time to complete.
 However, if your filesystem is quite small, then it will complete quite quickly. 
 The `-y` switch can be supplied to automate acceptance of user input.
 
+==== hsmdaemon
+
+`encryption:hsmdaemon` provides the ability to interact with an {hsm-overview-url}[hsm (Hardware Security Module)] which is integrated with ownCloud.
+
+NOTE: This command requires ownCloud to be integrated with xref:configuration/server/security/hsmdaemon/index.adoc[hsmdaemon], which requires ownCloud Enterprise Edition >= 10.2.
+For more information about hsmdaemon, please contact {owncloud-support-url}[ownCloud support].
+
+===== Usage
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} encryption:hsmdaemon [options]
+----
+
+===== Options
+
+[width="100%",cols="40%,60%",]
+|===
+| `--export-masterkey` 
+a| This command exports a base64-encoded representation of an ownCloud server’s master key private key file, which is used by its default encryption module.
```

Comment:
The master private key has the key ID which is there in the hsm device. May be there is a chance that user might get confused. Mind having a sentence which does not imply that user need not worry about exporting the data in the private key as it has only the id?

Reply:
So the user will never need to use this command?

Reply:
Or, is this command only meant for certain types of users?

Reply:
> is this command only meant for certain types of users?

I hope you are not getting confused with user-keys encryption. All I mean to say is this command would give the base64-encoded representation of the private key. And this private key (i.e, the one in the ownCloud instance and not in the hsm) has the id of the private key in th hsm device. Let me know if you still need clarification.

Reply:
Sorry for the slow delay on this. I'll get in touch if I need further information.

Reply:
@sharidas, this has been updated in ab86f07. I hope it's now correct. Please let me know if it is not.

Reply:
@sharidas, any further feedback?